LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright 2020 Scott Mackensen
```

Comment:
Might want to add MIT license at the top of the file, at least that's how I've seen other repos do it https://github.com/cmderdev/cmder/blob/master/LICENSE#L1
Not sure how Github parses the license but that might be how

Reply:
done